src/main/java/com/powsybl/openloadflow/util/Reports.java

```diff
@@ -44,85 +52,207 @@ public static void reportNetworkBalance(Reporter reporter, double activeGenerati
                 .withKey("mismatchDistributionSuccess")
-                .withDefaultMessage("Iteration ${iteration}: slack bus active power (${initialMismatch} MW) distributed in ${iterationCount} iterations")
-                .withValue(ITERATION, iteration)
+                .withDefaultMessage("Slack bus active power (${initialMismatch} MW) distributed in ${iterationCount} inner iteration" + plural)
                 .withTypedValue("initialMismatch", slackBusActivePowerMismatch, OpenLoadFlowReportConstants.MISMATCH_TYPED_VALUE)
                 .withValue("iterationCount", iterationCount)
                 .withSeverity(TypedValue.INFO_SEVERITY)
                 .build());
     }
 
-    public static void reportNoMismatchDistribution(Reporter reporter, int iteration) {
+    public static void reportPvToPqBuses(Reporter reporter, int pvToPqBusCount, int remainingPvBusCount) {
+        String defaultMessageBeginning = pvToPqBusCount == 1 ?
+                "${pvToPqBusCount} bus switched PV -> PQ " :
+                "${pvToPqBusCount} buses switched PV -> PQ ";
+        String defaultMessageEnd = remainingPvBusCount == 1 ?
+                "(${remainingPvBusCount} bus remains PV)" :
+                "(${remainingPvBusCount} buses remain PV)";
         reporter.report(Report.builder()
-                .withKey("NoMismatchDistribution")
-                .withDefaultMessage("Iteration ${iteration}: already balanced")
-                .withValue(ITERATION, iteration)
+                .withKey("switchPvPq")
+                .withDefaultMessage(defaultMessageBeginning + defaultMessageEnd)
+                .withValue("pvToPqBusCount", String.format("%6s", pvToPqBusCount))
+                .withValue("remainingPvBusCount", String.format("%6s", remainingPvBusCount))
                 .withSeverity(TypedValue.INFO_SEVERITY)
                 .build());
     }
 
-    public static void reportPvToPqBuses(Reporter reporter, int pvToPqBusCount, int remainingPvBusCount) {
+    public static void reportPqToPvBuses(Reporter reporter, int pqToPvBusCount, int blockedPqBusCount) {
+        String defaultMessageBeginning = pqToPvBusCount == 1 ?
+                "${pqToPvBusCount} bus switched PQ -> PV " :
+                "${pqToPvBusCount} buses switched PQ -> PV ";
+        String defaultMessageEnd = blockedPqBusCount == 1 ?
+                "(${blockedPqBusCount} bus blocked PQ due to the max number of switches)" :
+                "(${blockedPqBusCount} buses blocked PQ due to the max number of switches)";
         reporter.report(Report.builder()
-                .withKey("switchPvPq")
-                .withDefaultMessage("${pvToPqBusCount} buses switched PV -> PQ (${remainingPvBusCount} bus remains PV}")
-                .withValue("pvToPqBusCount", pvToPqBusCount)
-                .withValue("remainingPvBusCount", remainingPvBusCount)
+                .withKey("switchPqPv")
+                .withDefaultMessage(defaultMessageBeginning + defaultMessageEnd)
```

Comment:
same / don't do that, all reports with the same key MUST have the same defaultMessage

also in many other places below